model/metrics/gen-ai.yaml

```diff
@@ -0,0 +1,33 @@
+groups:
+  - id: metric_attributes.gen_ai
+    type: attribute_group
+    brief: 'LLM attributes'
+    attributes:
+      - ref: error.type
+        requirement_level:
+          conditionally_required: "if the operation ended in an error"
+      - ref: server.address
+        requirement_level: required
+      - ref: gen_ai.response.model
+        requirement_level: required
```

Comment:
presence of the response is not guaranteed (e.g. no response when connection error happens) and the attribute is recommended on spans so we can't make it required here.

We can 
1. add request model attribute as well
2. entertain the idea of having one model attribute that describes the 'best known' (the most specific one).

I'd start with option 1 in this PR and see if we get more evidence that option 2 is needed